student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
```

Comment:
- [ ] Please add `classroom` to the `initialize` method parameters.
It will be like that:
```ruby
 def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
Alright thanks, I will fix it right away.